tests/spec/core/structure-spec.js

```diff
@@ -52,17 +59,21 @@ describe("Core - Structure", () => {
     };
     ops.config.tocIntroductory = true;
     const doc = await makeRSDoc(ops);
-    const $toc = $("#toc", doc);
-    expect($toc.find("h2").text()).toEqual("Table of Contents");
-    expect($toc.find("> ol > li").length).toEqual(6);
-    expect($toc.find("li").length).toEqual(18);
+    const toc = doc.querySelector("#toc");
+    expect(toc.querySelector("h2").textContent).toEqual("Table of Contents");
+    const heirLI = [...utils.children(toc, "ol > li")];
+    expect(heirLI.length).toEqual(6);
+    expect(toc.querySelectorAll("li").length).toEqual(18);
+    expect(heirLI[0].textContent).toEqual("Abstract");
     expect(
-      $toc
-        .find("> ol > li a")
-        .first()
-        .text()
-    ).toEqual("Abstract");
-    expect($toc.find("> ol > li a[href='#intro']").length).toEqual(1);
+      heirLI.reduce(
```

Comment:
Maybe we can do:

``` js
(result, element) => result.concat[[...element.querySelectorAll("a[href='#intro']")]]
```

Reply:
Or we can do `utils.children(toc, "ol > li a[href='#intro']")`. It will cause some overhead but should be insignificant.

Reply:
Didn't see your comment 😅